### DISCUSSION TITLE
Feb 6

### DISCUSSION THREAD

**Discussion by ngsy** [GymHubCommunity/GymHub-FE#119] [DailyMeeting]
### ⚽️ 회의 내용

1. aws 공통 계정 생성
  - 프론트 측에서 생성해서 공유하기
  -슬희님 당첨  aws 공통계정 생성 
2. 이미지 업로드 방식 결정
  - presigned url 을 활용해서 프론트에서 처리  == 프론트 -> 서버 (이걸로 결정!) 
  - 백에서 처리  == 프론트  -> 서버 -> 프론트 -> 서버 
3. 디자인 피드백 
  - 슬희: 너무 이쁘다 다른팀 디자인보다 훠어어얼씬 낫다. 헬스 컨셉이 조금 더 들어났으면 좋겠다. 
  - 수 : 헬스컨셉이 조금 더 들어갔으면 좋겠다, 하트가 덤벨로 바겼으면 좋겠다. 운동 커밋도 커밋색깔이 여러가지라 이해하기 어렵다. 한가지색상으로 통일 
    했으면 좋겠다. , 인스타랑 너무 비슷하다. 
    회원가입이 너무 심플하다. 
- 수영 : 잔디가 너무 똑같아서 아쉽다. 
- 윤혁 : 운동기록 게시글이 너무 보기 힘들다. 

개선점 
 - 회원가입에서 귀여운 캐릭터가 나와서 설명해주면 좋겠다. 
 - 하트가 덤벨로 바꼈으면좋겠다. 
 - 헬스 잔디 : 주 , 월 단위가 있었으면 좋겠다. 
 - 헬스 컨셉이 들어갔으면 좋겠다. 
 - 인스타 복붙느낌이라, 차별점이 있었으면 좋겠다. 
 
5. 개발 현황 공유 
수 : 온보딩 페이지 , 로그인 페이지까지 디자인 싹다 만들예정 다죽었어.
윤혁 : 게시글 상세화면 , 게시글 목록 , 다 비슷해서 와이어프레임으로는 올려놓긴하였다. 디자인 확정나면 수정하겠다. mock데이터를 사용 해보셔야할듯
슬희 : text 컴포넌트를 공통으로 해보려 하고있다. scss로 공통화 하는게 잘 모르겠다. classNames 사용하시면 됩니다. 
수영  : 글쓰기 와이어프레임 진행하고있습니다. 

용어 정리 
게시글 : Post 
게시글 목록 : Feed 

### 🏀 오늘 할 일

수🔥 :   개발현황과 동일 진행
윤혁❤️ : 게시글 상세화면이 나왔으니 mock데이터를 만들어보겠다. presigned url
슬희 : text 컴포넌트를 공통화 처리하고있다. 
수영  : 개발현황 동일 진행

### 🎱 TMI

수  : 굿굿 PT 환불 받으셨다.달리기 
윤혁 : 마라톤 고민좀 해볼게요, 집중이 잘안된다.
슬희 : 프론트는 너무쉽다 백이 어렵다 백잘하는 사람은 프론트도 잘한다. 내일 위워크 갈 예정 
수영  : 오늘 영화관에서 축구봅니다! 후후후후

**Comment by hanseulhee** [comment]
제 발언 아닙니다 !!! 프론트 최고 ~
